### ISSUE TITLE
DeprecationWarning: The verify parameter is deprecated. Please use options instead.

### ISSUE THREAD

**Issue by fabrizio2210** [vimalloc/flask-jwt-extended#366]
The code line 91 in the file [ask_jwt_extended/utils.py](https://github.com/vimalloc/flask-jwt-extended/blob/master/flask_jwt_extended/utils.py)
```
    jwt_manager = _get_jwt_manager()
    unverified_claims = jwt.decode(
        encoded_token, verify=False, algorithms=config.decode_algorithms
    )
```

triggers a deprecation warning in jwt/api_jws.py:119 saying:
```
venv/lib/python3.7/site-packages/jwt/api_jws.py:119: DeprecationWarning: The verify parameter is deprecated. Please use options instead.
  'Please use options instead.', DeprecationWarning)
```

**Comment by vimalloc-mavenlink** [comment]
Thanks for the report. I'll get this fixed up. It looks like PyJWT just released a 2.0.0 alpha build, so I'll see about upgrading to that when it stabilizes at the same time :+1:

**Comment by vimalloc** [comment]
Current version will now require a version of pyjwt lower then 2.0, and the 4.0.0 branch has things updated to pyjwt 2.0.0 :+1:
